infrastructure/tooling/templates/k8s/deployment.yaml

```diff
@@ -55,23 +55,29 @@ spec:
               port: 80
             timeoutSeconds: 5
             periodSeconds: 10
-            initialDelaySeconds: 3
-        # After 24 hours, we start randomly deciding to exit the container.
-        # Taskcluster services are created with the assumption that they will
-        # not run indefinitely.
+            initialDelaySeconds: 6
         livenessProbe:
-          exec:
-            command:
-            - /bin/sh
-            - -c
-            - exit $(awk 'BEGIN{srand(); print (rand() > 0.3)}')
-          initialDelaySeconds: 86400
-          periodSeconds: 60
-          failureThreshold: 1
+          $if: 'needsService'
+          then:
+            httpGet:
+              path: ${readinessPath}
+              port: 80
+            timeoutSeconds: 3
+            initialDelaySeconds: 15
+            periodSeconds: 30
+          else:
+            exec:
+              command:
+              - /bin/sh
+              - -c
+              - exit $(awk 'BEGIN{srand(); print (rand() > 0.3)}')
+            initialDelaySeconds: 432000 # 5 days
```

Comment:
this is to experiment if we need this for background jobs, i.e. they leak memory or run slower after 1 day. Bump from 1d to 5d. 
Kinda to be safe, but maybe not necessary. We'll see